gleam.toml

```diff
@@ -1,6 +1,8 @@
 name = "code_review"
 version = "1.0.0"
 
+gleam = ">= 1.1.0 and < 2.0.0"
```

Comment:
What about the project requires 1.1.0?

Reply:
I'd probably no limit it to under 2

Reply:
Hmmm technically there is nothing forcing this to only use gleam >= 1.1.0.  We can probably leave this one unchanged and I can definitely remove the less than 2.0 constraint

Reply:
fixed :D thanks for the feedback folks